install.md

```diff
@@ -156,6 +187,24 @@ option.  Below are the commands for Conda.
      scripts). If no such modification is required, using `pip install .` is 
      sufficient.
 
+    + Install `element-interface`
+        + `element-interface` contains the scripts to load data for 
+        `element-array-ephys` and `workflow-array-ephys`.
+
+        + `element-interface` is a dependency of `element-array-ephys` and
+         `workflow-array-ephys`, however it is not contained within `requirements.txt`.
```

Comment:
I'm seeing `element-interface` as a git install requirement in [the latest workflow-array-ephys PR](https://github.com/datajoint/workflow-array-ephys/blob/59ee47d6816ccca40cbe1063d9ecdaefb7986742/requirements.txt#L6). Does that change what our wording should be here? See also line 246 of this file, same 'not contained within' text.

Reply:
Thanks for the fix with https://github.com/kabilar/datajoint-elements/pull/1.